client/qubesdb-cmd.c

```diff
@@ -253,6 +262,7 @@ static void usage(char *argv0) {
     fprintf(stderr, "  -d <domain> - specify destination domain, available only in dom0\n");
     fprintf(stderr, "  -w - wait for any value (possibly empty)\n");
     fprintf(stderr, "  -x - remove the value after reading it (affects reading commands)\n");
+    fprintf(stderr, "  --default <default> - use <default> if the value does not exist\n");
```

Comment:
Ah, indeed `-d` is used already. But I'd still like this to have some short option too, can be `-e` in this case.